cherryontech/JourneyWise
# Archetype Picture Fix, Email Confirmation, Homepage Styling

## Pull Request

### Description
Attempt to fix the archetype pictures so they show up in Netlify, styled the email confirmation and added a progress spinner, fixed some of the homepage text styling. 

### Related Issue
???

### Description of Changes
* Switched the src of the archetype pictures to public in hopes that they show up in Netlify. 
* Styled the email confirmation based on the designs. 
* Added a circular progress spinner so users can see something happening while the email is being sent (before there was a super awkward pause where you aren't sure if anything is happening). 
* Fixed up margins and things on the homepage so the text looks more like the design. I think this text won't move! 

### Checklist
- [ ] I have updated the documentation or comments where necessary.

### Testing Instructions
* N/A 
* But please check to see if the archetype photos show up in the questionnaire! 

### Screenshots (if applicable)


### Gotchas/Learnings
* N/A

### Additional Notes (Optional)
* N/A
